kong/api/routes/upstreams.lua

```diff
@@ -118,7 +118,7 @@ return {
     end,
   },
 
-  ["/upstreams/:upstream_name_or_id/targets/active/"] = {
```

Comment:
I believe (because I tested this yesterday) that removing this makes the endpoint works with both a trailing slash and without. We should also write a new regression test that ensures it is the case (after all, this is a regression we don't want to see happening again in the future).

Reply:
yep, good idea